models/articles.models.js

```diff
@@ -27,6 +27,17 @@ exports.fetchArticles = async () => {
     GROUP BY articles.article_id
     ORDER BY created_at DESC 
     `)
-    console.log(queryResponse.rows)
+    return queryResponse.rows
+}
+
+exports.fetchArticleComments = async (article_id) => {
+    const queryResponse = await db.query(`
+    SELECT * FROM comments
+    WHERE article_id = $1
+    ORDER BY created_at DESC
+    `, [article_id])
+    if (queryResponse.rows.length === 0) {
```

Comment:
Empty rows of comments. Does it mean no article with that it? Or does it mean no comments for the existing article yet?